docs/cinder-csi-plugin/examples.md

```diff
@@ -254,3 +255,103 @@ $ openstack volume list
 | 07522a3b-95db-4bfa-847c-ffa179d08c39 | pvc-400b1ca8-8786-435f-a6cc-f684afddbbea | available |    1 |                                                 |
 | bf8f9ae9-87b4-42bb-b74c-ba4645634be6 | pvc-4699fa78-4149-4772-b900-9536891fe200 | available |    1 |                                                 |
 ```
+
+## Disaster recovery of PV and PVC
+
+This example assume your Kubernetes cluster is crashed, but your volumes
+are still available in the OpenStack cloud:
```

Comment:
curious, if the original cluster is down, how can we create the pv/pvc yaml file? we have enough info?

Reply:
I think you need the volume IDs from OpenStack and puzzle the yaml files together. A backup of this files with etcd backup, rancher-backup, kube-backup would also help.

Reply:
I think the info you provided above might be helpful in the doc as well.. wdyt?

Reply:
good idea, have added some notes